Makefile

```diff
@@ -25,6 +25,11 @@ test: test-unit
 test-unit: clean
 	mvn test
 
```

Comment:
Do we want test-integration to be called as part of make test along with test-unit?